docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: page
+title: Alluxio
+nav_order: 5
+parent: Getting-Started
+---
+# Get Started with RAPIDS with Alluxio
+
+RAPIDS plugin can remarkably accelerate the computing part of the whole SQL query by
+leveraging GPUs, but it’s hard to accelerate the data reading process when the data
+is in the cloud file system because of network overhead. Alluxio is an open source
+data orchestration platform that brings your data closer to compute across clusters,
+regions, clouds, and countries for reducing the network overhead. This guide will
+go through how to set up the RAPIDS Accelerator for Apache Spark with Alluxio with
+a on premise cluster.
+
+## Prerequisites
+
+Assuming that user has successfully setup and ran the RAPIDS Accelerator with on premise
+cluster according to [this doc](getting-started-on-prem.md)
+
+This guide will deploy Alluxio on Yarn cluster with 2 NodeManagers and 1 ResourceManager,
+and describe the instructions to configure Swiftstack as Alluxio’s under storage system.
+
+Let's assume the hostnames of Yarn cluster are respectively
+
+``` json
+RM_hostname
+NM_hostname_1
+NM_hostname_2
+```
+
+## Alluxio setup
+
+It is recommending to deploy Alluxio workers in each NodeManager and Alluxio master in
+ResourceManager.
+
+1. Download the latest Alluxio version (2.4.1) **alluxio-${LATEST}-bin.tar.gz** from [alluxio website](https://www.alluxio.io/download/).
+2. Copy `alluxio-${LATEST}-bin.tar.gz` to all the NodeManagers and ResourceManager.
+3. Extract `alluxio-${LATEST}-bin.tar.gz` to the directory specified by **ALLUXIO_HOME**
+   in NodeManager and ResourceManager.
+
+   ``` shell
+   # Let's assume to extract alluxio to /opt
+   mkdir -p /opt
+   tar xvf alluxio-${LATEST}-bin.tar.gz -C /opt
+   export ALLUXIO_HOME=/opt/alluxio-${LATEST}
+   ```
+
+4. Configure alluxio.
+   - Alluxio master configuration
```

Comment:
we should put something here that says see section below this for explanation of the settings and verify they are right for your environment.
Also this is really configuration for both master and workers right?